x/docs/alias.go

```diff
@@ -19,6 +19,9 @@ var (
 )
 
 type (
-	Keeper           = keeper.Keeper
-	MsgShareDocument = types.MsgShareDocument
+	Keeper             = keeper.Keeper
+	Document           = types.Document
+	DocumentReceipt    = types.DocumentReceipt
+	MsgShareDocument   = types.MsgShareDocument
+	MsgDocumentReceipt = types.MsgSendDocumentReceipt
```

Comment:
I think we should rename the alias as well, what do you think? This would make the names clearer